### ISSUE TITLE
Focus and screen-reader issues with Help pane secondary toolbar

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6091]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.777
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- <kbd>Ctrl+3</kbd> to display and focus the Help pane
- keyboard focus is on the frame containing help content and you can navigate this via keyboard like any web page (<kbd>Tab</kbd>, <kbd>Shift+Tab</kbd>)
- (problem one) <kbd>Shift+Tab</kbd> to the beginning of the help frame, then <kbd>Shift+Tab</kbd> again
- (problem two) Get focus to the primary Help toolbar (hard to do, so cheat and use the mouse by clicking in the search control to the left of the refresh control), then <kbd>Tab</kbd> off the end of that toolbar
- (problem three) Click in the secondary toolbar's "Find in topic" control to give it focus, then type something that causes the next/previous buttons to appear next to it; then hit <kbd>Tab</kbd>
- (problem four) Click in the secondary toolbar's "Find in topic" control to give it focus, then <kbd>Shift+Tab</kbd>
- (problem five) Use a screen reader and navigate the cursor to the dropdown control in the secondary toolbar. Attempt to understand and use it without visually seeing it.

### Describe the problem in detail
- (problem one) focus briefly bounces to the "Find in topic" edit box in the secondary toolbar, then immediately bounces back to the frame
- (problem two) focus briefly goes to the secondary toolbar's "Find in topic" control, skipping over the dropdown at the start of the toolbar, then bounces to the help frame
- (problem three) focus doesn't go to the next control ("<" button), but to the help frame
- (problem four) focus doesn't go to (what looks like a) dropdown control to the left of the "Find in topic" field, but bounces to the help frame
- (problem five) This "control" is not presented to screen readers as a dropdown menu. It shows up as a static label (which happens to accept a click but doesn't announce that it accepts a click), and an unlabelled image (the dropdown arrow). Neither accept keyboard focus. If you activate with screen-reader, the dropdown menu shows but isn't announced so you don't know anything happened.

### Describe the behavior you expected
In all cases, keyboard focus should behave "normally"; progress left to right, top to bottom, when entering and exiting the secondary toolbar from either direction.

The dropdown control should be annotated correctly so a screen-reader user can operate it.

<img width="534" alt="screen shot of RStudio help pane showing the toolbars and part of the content frame" src="https://user-images.githubusercontent.com/10569626/73107863-c269c080-3eb3-11ea-979d-ed14be55a29c.png">
